runtime/flangrti/CMakeLists.txt

```diff
@@ -91,6 +91,9 @@ if (NOT DEFINED LIBOMP_EXPORT_DIR)
     libomp.so
     HINTS ${CMAKE_BINARY_DIR}/lib)
   target_link_libraries(flangrti_shared ${FLANG_LIBOMP})
+else()
+  link_directories(${LIBOMP_EXPORT_DIR})
+  target_link_libraries(flangrti_shared omp)
```

Comment:
Could you explain this change?

Reply:
Add openmp library link custom directories.

Is it better to split the openmp part to another patch?

Thanks,
Leslie Zhai

Reply:
Yes, if it is addressing a different issue than a separate patch is better.

BTW, may i ask what is your interest in Flang? Is it hobby or professional work?

Reply:
Hobby :)

I [migrated dragonegg](https://reviews.llvm.org/D35667) .  And [Chad Rosier](https://reviews.llvm.org/p/mcrosier/) introduced flang.

Thanks,
Leslie Zhai